src/balena.sbvr

```diff
@@ -545,6 +546,9 @@ Fact type: device is managed by service instance
 Fact type: device should be running release
 	Synonymous Form: release should be running on device
 	Necessity: each device should be running at most one release
+Fact type: device should have hostapp release
```

Comment:
i'm 100% open to better names for this.

Reply:
Just rubber ducking as I don't have anything to suggest, but how would you describe a hostapp and what does it mean for a device to run one of its releases?

Reply:
I also don't have a better suggestion, but a hostapp is just an image that has some scaffolding so that mobynit can find it, mount it and pivot root into it to launch systemd. I wouldn't say it's a bootable image, as that implies bootloaders and firmware more than mobynit, but maybe mobynit bootstrapped image or similar.

Reply:
Strawman:

```
Fact type: device is bootstrapped by release
```

Sidenote: the model should enforce that this release is of a hostapp.

Reply:
those suggestions make more sense than what I have! thanks for thinking through it. we do enforce that the related release belongs to a hostapp here: https://github.com/balena-io/open-balena-api/pull/634/files#diff-ab9e54b98b8703135b751e1127edf11c3c5e3b48e3e9892d8acc07c12767ecb5R786

Reply:
I went with `Fact type: device is initialized by release` as it felt more natural to me, thanks again for the input!